drivers/spi/spi_context.h

```diff
@@ -147,32 +147,22 @@ static inline void spi_context_complete(struct spi_context *ctx, int status)
 #endif /* CONFIG_SPI_ASYNC */
 }
 
-static inline int spi_context_cs_active_value(struct spi_context *ctx)
+static inline u32_t spi_context_cs_active_level(struct spi_context *ctx)
 {
 	if (ctx->config->operation & SPI_CS_ACTIVE_HIGH) {
-		return 1;
+		return GPIO_ACTIVE_HIGH;
 	}
 
-	return 0;
-}
-
-static inline int spi_context_cs_inactive_value(struct spi_context *ctx)
-{
-	if (ctx->config->operation & SPI_CS_ACTIVE_HIGH) {
-		return 0;
-	}
-
-	return 1;
+	return GPIO_ACTIVE_LOW;
 }
 
 static inline void spi_context_cs_configure(struct spi_context *ctx)
 {
 	if (ctx->config->cs && ctx->config->cs->gpio_dev) {
+		u32_t flags = spi_context_cs_active_level(ctx);
 		gpio_pin_configure(ctx->config->cs->gpio_dev,
-				   ctx->config->cs->gpio_pin, GPIO_OUTPUT);
-		gpio_pin_set(ctx->config->cs->gpio_dev,
 			     ctx->config->cs->gpio_pin,
-			     spi_context_cs_inactive_value(ctx));
+			     flags | GPIO_OUTPUT_INACTIVE);
```

Comment:
indentation is wrong

Reply:
Until #21392 gets settled we don't really have a way to determine what "proper indentation" is, and existing code throughout the baseline varies on where alignment of the non-whitespace wrapped content occurs.

That issue is likely to make progress in the 2.5 or 2.6 development stages.

Reply:
Zephyr uses linux code style (plus some extra). Proper editors can apply it. But indeed a automatic verification would be the best.